tests/zkasm/run-tests-zkasm.js

```diff
@@ -10,6 +10,7 @@ const {
     newCommitPolsArray
 } = require('pilcom');
 const buildPoseidon = require('@0xpolygonhermez/zkevm-commonjs').getPoseidon;
+const assert_helper = require('./assert');
```

Comment:
Let's use `AssertHelper` name here instead of `assert_helper` to signal that this is a class. This will also make it easier to search the codebase as this name will match the one in the `assert.js`.